spec/test-data/translations/glotpress-download/stubs/fr.xml

```diff
@@ -54,4 +56,33 @@ Language: fr
         <item> - o Deux: deuxième point</item>
     </string-array>
     <string name="multi_range_statement">La première plage est 2-100 et la seconde est 123-456.</string>
+    <string name="sunday">Dimanche</string>
+    <string name="monday">Lundi</string>
+    <string name="tuesday">Mardi</string>
+    <string name="wednesday">Mercredi</string>
+    <string name="thursday">Jeudi</string>
+    <string name="friday">Vendredi</string>
+    <string name="saturday">Samedi</string>
+    <string-array name="weeks_full">
+        <item>\@string/sunday</item>
+        <item>\@string/monday</item>
+        <item>\@string/tuesday</item>
+        <item>\@string/wednesday</item>
+        <item>\@string/thursday</item>
+        <item>\@string/friday</item>
+        <item>\@string/saturday</item>
+        <item>Aucun</item>
+    </string-array>
+    <string-array name="progress" formatted="false">
+        <item>0%%</item>
+        <item>25%</item>
+        <item>50%%</item>
+        <item>75%</item>
+        <item>100%</item>
+    </string-array>
```

Comment:
Intentional use of `%%` on a `formatted=false` resource of type `string-array` in this GlotPress export stub, so we could write our test on `quick_lint` to validate it'd emit a warning as appropriate.